ReactWindows/ReactNative.Shared/PlatformServices/ServiceResolver.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace ReactNative.PlatformServices
+{
+    /// <summary>
+    /// Singleton for platform-specific service provider.
+    /// </summary>
+    public static class ServiceResolver
+    {
+        private static IServiceProvider s_current = new CurrentPlatformServiceProvider();
```

Comment:
Eventually we'll need to eliminate this and hook the implementation into some other ReactNative initialization phase. It will be nice because we can replace initialization for dependency injection, ReactChoreographer, dispatcher helpers, etc. with a single init call.